modules/nw-ovn-kubernetes-observability.adoc

```diff
@@ -0,0 +1,172 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/ovn-kubernetes-troubleshooting-sources.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ovn-kubernetes-observability_{context}"]
+= Checking OVN-Kubernetes network traffic with OVS sampling using the CLI
+:FeatureName: Checking OVN-Kubernetes network traffic with OVS sampling
+include::snippets/technology-preview.adoc[]
+
+OVN-Kubernetes network traffic can be viewed with OVS sampling via the CLI for the following network APIs: `NetworkPolicy`, `AdminNetworkPolicy`, `BaselineNetworkPolicy`, `UserDefinesdNetwork` isolation, `EgressFirewall`, and multicast ACLs. Scripts for these networking events are found in the `/usr/bin/ovnkube-observ` path of each OVN-Kubernetes node.
+
+Unlike the Network Observability Operator, which is intended for observability and debugging, checking OVN-Kubernetes network traffic with OVS sampling using the CLI is intended to help with packet tracing and debuggability. It can be used while the Network Observability Operator is installed, however that is not a requirement.
```

Comment:
s/debugging/debugability

I think you don't lose the intended meaning with _debugging_ and it is a bit simpler.

Reply:
The first sentence is a touch confusing, because at the start, I think you are going to make a contrast between the Network Observabilty Operator and checking OVN-kube network traffic. But I think you are comparing and contrasting, right? Both are useful for debugging, but the Operator is good for observabilty and checking the network traffic is good for packet tracing.

Anyhow, just a minor comment.